docs/swift-sdk/integrate-with-okta.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Integrate With Okta
+sidebar_position: 2
+---
+
+This guide describes how to configure Beyond Identity using Okta as the OIDC authentication flow. 
+
+### Prerequisites
+
+ - [Integrate With Okta](../sso-integrations/integrate-with-okta)
+ - [Swift SDK Overview](overview)
+
+Before calling [`Embedded.shared.authenticate`](#authorize-with-okta), we must [Authenticate With Okta](integrate-with-okta#authorize-with-okta)
+
+### Authorize With Okta
+
+ - Step 1: Okta URL
+
+To begin the authentication flow, start an `ASWebAuthenticationSession`, and load the OAuth2 authorization request URL provided by Okta.
+
+```swift
```

Comment:
I'm not sure why, but for some reason, syntax highlighting doesn't work with `swift` or `kotlin`. There's probably some setting we need to modify with docusaurus, but in the meantime, I've been defaulting to javascript for everything just to get any sort of syntax highlighting.